### PR TITLE
Reformat LOCA grid cell title

### PR DESCRIPTION
**Pull request by elehmer**
This PR adapts the website to the change in the LOCA grid cell title format from:

`LOCA Grid Cell %longitude%, %latitude%` to `LOCA Grid Cell %latitude%, %longitude%`

### CODE REVIEW

File: src/helpers/geocode.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ export const searchBoundaryLayer = async (searchStr, boundaryId) => {
   const searchParams = {
     srs: 4326,
     search: searchStr,
-    v: 1, // cache bust 2022-05-09
```
Comment (mukhtyar) [inline src/helpers/geocode.js:71]: An optional update is to create a new `const v = 2` at the top of this file and then use it for three other places it appears.

**Review by mukhtyar — APPROVED** [review]

File: src/helpers/geocode.js (hunk 1 above)
Comment (elehmer) [reply src/helpers/geocode.js:71]: > An optional update is to create a new `const v = 2` at the top of this file and then use it for three other places it appears.

True, that would be more efficient wouldn't it. I'll make the changes...

**Comment by elehmer** [comment]
Better?

File: src/helpers/geocode.js (hunk 1 above)
Comment (elehmer) [reply src/helpers/geocode.js:71]: Better?

**Comment by elehmer** [comment]
[Add hemisphere notation to lat/long display for LOCA grid cells](https://trello.com/c/R2ik2vrt/212-add-hemisphere-notation-to-lat-long-display-for-loca-grid-cells)

**Pull request merged**